EIPS/eip-7201.md

```diff
@@ -4,7 +4,8 @@ title: Namespaced Storage Layout
 description: A formula for the storage location of structs in the namespaced storage pattern.
 author: Francisco Giordano (@frangio), Hadrien Croubois (@Amxx), Ernesto García (@ernestognw), Eric Lau (@ericglau)
 discussions-to: https://ethereum-magicians.org/t/eip-7201-namespaced-storage-layout/14796
-status: Review
+status: Last Call
+last-call-deadline: 2023-10-01
```

Comment:
```suggestion
last-call-deadline: 2023-10-09
```

Reply:
As mentioned earlier (https://github.com/ethereum/EIPs/pull/7519#discussion_r1318832079) I didn't want to set the deadline beyond Oct 1st because we're doing a release that day and after that point as authors we won't accept normative changes to the EIP.